src/Storage/Storage/Common/Util.cs

```diff
@@ -419,6 +419,25 @@ public static BlobBaseClient GetTrack2BlobClient(BlobContainerClient track2conta
             return GetTrack2BlobClient(blobUriBuilder.ToUri(), context, options, blobType);
         }
 
+        public static BlobBaseClient GetTrack2BlobClientNoSlashTrim(BlobContainerClient track2container, string blobName, AzureStorageContext context, string versionId = null, bool? IsCurrentVersion = null, string snapshot = null, BlobClientOptions options = null, global::Azure.Storage.Blobs.Models.BlobType? blobType = null)
+        {
+            //Get Track2 Blob Client Uri
+            BlobUriBuilder blobUriBuilder = new BlobUriBuilder(track2container.Uri, false)
```

Comment:
I have another idea, we might don't need a new function, but just add a new parameter like "TrimBlobNameSlashes = true" as the last parameter to the orignal function:
1. Don't need dup code
2. Other place call this function don't need change, since the default is still true
3. For the list blob, call the function by adding parameter "TrimBlobNameSlashes: true"

How do you think about it?

Reply:
Yes added.